planning/behavior_path_planner/include/behavior_path_planner/utils/avoidance/utils.hpp

```diff
@@ -94,6 +94,10 @@ void fillObjectMovingTime(
   ObjectData & object_data, ObjectDataArray & stopped_objects,
   const std::shared_ptr<AvoidanceParameters> & parameters);
 
+void fillObjectStoppableJudge(
```

Comment:
it's better those kind of explanation is described(I don't describe well though...)

/// @brief Determines if the object can be stopped or not based on hogehoge
/// @param[in,out] object_data ObjectData structure that represents the object for which we need to determine if it can be stopped
/// @param[in] registered_objects Array of registered objects
/// @param[in] feasible_stop_distance Feasible stopping distance calculated based on the hoge
/// @param[in] parameters Avoidance parameters

Reply:
I'll add doxygen style description for all utils in other PRs. Thanks!